corehq/apps/hqmedia/static/hqmedia/js/reference_controller.js

```diff
@@ -13,37 +13,65 @@ hqDefine("hqmedia/js/reference_controller",[
 ) {
     var HQMediaUploaders = mediaUploaders.get();
 
-    function MultimediaReferenceController(options) {
-        assertProperties.assert(options, ['references', 'objectMap', 'totals']);
+    function MultimediaReferenceController() {
         var self = {};
-        self.objectMap = options.objectMap;
         self.references = ko.observableArray();
         self.showMissingReferences = ko.observable(false);
-        self.totals = ko.observable(options.totals);
+        self.totals = ko.observableArray();
```

Comment:
totals? is that just like `references`? that name is a bit confusing

Reply:
Not exactly: `references` is a list that may contain duplicate files (if, say, the same image is used as an icon for two different forms). `totals` is a dict that contains the number of unique items for each type (image, audio, video) - it's what's used to populate the "19/20 Image references"-type text displayed with the download zip button.

`totals` => `unique_item_counts_by_type`?

Reply:
I took a pass at renaming, but it touches more code than I'd like to update here - the backend also uses "totals" a number of places. Going to leave this as-is for now.